.travis.yml

```diff
@@ -13,15 +13,15 @@ matrix:
     - php: 5.6
     - php: hhvm
     - php: 7.0
-    - php: 7.1
-    - php: 7.1
+    - php: nightly
+    - php: nightly
       env: DEPENDENCIES='dev'
```

Comment:
dev deps should be tested on a stable PHP version IMO, to avoid having this job failing due to PHP issues